.fixtures.yml

```diff
@@ -1,9 +0,0 @@
-fixtures:
```

Comment:
We need this for spec tests

Reply:
see commit message. this is only stuff i needed to remove for making it work on my system. will readd it after finished.